packages/lesswrong/components/common/CloudinaryImage2.tsx

```diff
@@ -20,46 +23,74 @@ function cloudinaryPropsToStr(props) {
   return sb.join(",");
 }
 
-// Cloudinary image without using cloudinary-react. Allows SSR. See:
-// https://github.com/LessWrong2/Lesswrong2/pull/937 "Drop cloudinary react"
-// https://github.com/LessWrong2/Lesswrong2/pull/964 "Temporarily revert removal of cloudinary-react"
-const CloudinaryImage2 = ({width, height, objectFit, publicId, imgProps, className}: {
+function makeCloudinaryImageUrl (publicId: string, cloudinaryProps: CloudinaryPropsType) {
+  return `https://res.cloudinary.com/${cloudinaryCloudNameSetting.get()}/image/upload/${cloudinaryPropsToStr(cloudinaryProps)}/${publicId}`
+}
+
+// Cloudinary image without using cloudinary-react. Allows SSR.
+const CloudinaryImage2 = ({width, height, objectFit, publicId, imgProps, fullWidthHeader, className}: {
+  /** Overridden if fullWidthHeader is true */
   width?: number|string,
   height?: number,
-  objectFit?: string,
+  objectFit?: 'fill' | 'contain' | 'cover' | 'none' | 'scale-down',
   publicId: string,
   imgProps?: CloudinaryPropsType,
+  /** Overrides width */
+  fullWidthHeader?: boolean,
   className?: string,
 }) => {
-  const cloudinaryCloudName = cloudinaryCloudNameSetting.get()
-
-  let cloudinaryProps: any = {
+  let cloudinaryProps: CloudinaryPropsType = {
     c: "fill",
     dpr: "auto",
-    g: "custom",
```

Comment:
I removed this because it was there to try to get the cropping applied by the cropping tool to do anything, and it does not do anything. Revert the change to allow cloudinary's auto-gravity to save us from terrible cut-off images.